tests/epics/demo/test_demo.py

```diff
@@ -170,20 +170,20 @@ async def test_mover_disconncted():
     with pytest.raises(NotConnected):
         async with DeviceCollector(timeout=0.1):
             m = demo.Mover("ca://PRE:", name="mover")
-    assert m.name == "mover"
+        assert m.name == "mover"
```

Comment:
These should be in their original position

Reply:
pyright did not like that. it said that m may not be instantiated. 
Looking at it again I see that this is not going to behave as required with my change (because the expected exception would jump over final assert)
pyright is correct in that if the DeviceCollector constructor throws NotConnected then m would not be set.

But this is rather pedantic and should probably be ignored.

Reply:
or pre declare m.